ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/ipv4-acls.yml

```diff
@@ -0,0 +1,157 @@
+hostname: ipv4-acls
```

Comment:
do you want to sneak in the gitattribute thing for structured_config we talked about?

Reply:
nope